README.md

```diff
@@ -3,6 +3,11 @@
 ![Build status](https://github.com/guardian/sbt-riffraff-artifact/actions/workflows/ci.yml/badge.svg?branch=main)
 [![sbt-riffraff-artifact Scala version support](https://index.scala-lang.org/guardian/sbt-riffraff-artifact/sbt-riffraff-artifact/latest-by-scala-version.svg?targetType=Sbt)](https://index.scala-lang.org/guardian/sbt-riffraff-artifact/sbt-riffraff-artifact)
 
+**This project is DEPRECATED and you should use
```

Comment:
Should we use the [`>**Note**` notation](https://github.com/orgs/community/discussions/16925) to add more emphasis to this message?

Reply:
done!